src/drivers/display/fb/lv_linux_fbdev.c

```diff
@@ -249,14 +253,26 @@ void lv_linux_fbdev_set_force_refresh(lv_display_t * disp, bool enabled)
  *   STATIC FUNCTIONS
  **********************/
 
+static void write_to_fb(lv_linux_fb_t * dsc, uint32_t fb_pos, const void * data, size_t sz)
+{
+#if LV_LINUX_FBDEV_MMAP
+    uint8_t * fbp = (uint8_t *)dsc->fbp;
+    lv_memcpy(&fbp[fb_pos], data, sz);
+#else
+    pwrite(dsc->fbfd, data, sz, fb_pos);
```

Comment:
Add return value checking and error `errno` printing.

Reply:
Done.